datasafe-rest-impl/src/main/java/de/adorsys/datasafe/rest/impl/controller/InboxController.java

```diff
@@ -78,6 +101,11 @@ public void readFromInbox(@RequestHeader String user,
      * Deletes file from users' INBOX.
      */
     @DeleteMapping("/{path:.*}")
+    @ApiOperation("Delete document from inbox")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Document successfully deleted"),
+            @ApiResponse(code = 403, message = "Access denied")
```

Comment:
403 has default message, not necessary to add it

Reply:
removed